Dockerfile

```diff
@@ -1,28 +1,38 @@
 FROM ubuntu:18.04
 FROM httpd:2.4
+FROM python:3
```

Comment:
@pmazumda Python3 is already getting downloaded by the `RUN apt-get -y install python3-pip` line below. This layer might not be needed.